applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c

```diff
@@ -73,11 +73,56 @@ static void bond_find(const struct bt_bond_info *info, void *user_data)
 	}
 }
 
+static int addr_print(bt_addr_le_t const *const local_addr,
+		      bt_addr_le_t const *const directed_to_addr)
+{
```

Comment:
Might be good with some doc for this function. The directed_to_addr could probably be changed, it's hard to know that this is specifically for directed advertisement. Suggestion: ``dir_adv_addr`` or something similar, the adv should at least be in there somehow